background/lookup.js

```diff
@@ -18,6 +18,9 @@
 
 import { keccak_256 as sha3 } from 'js-sha3';
 import { omitBy } from 'lodash';
+import ParityLookup from 'lookup';
+import { getTransport } from './transport';
+import { Api } from '@parity/parity.js';
 
 const LOOKUP_STORAGE_KEY = 'parity::lookup_cache';
 // Time To Live for Lookup data (in ms : 1 day)
```

Comment:
I think that it might actually be a bit too much now. If your node is disconnected we may cache errors, and in case of errors it should retry earlier.

Second use case: if you register your e-mail you are most probably expecting to see the changes right after the transaction is mined.

I think it would be best to either lower TTL to an hour or split the TTLs for success responses and errors/nomatches and retry the latter more often.